devimint/src/federation.rs

```diff
@@ -212,11 +213,18 @@ impl Federation {
             );
             let admin_client =
                 DynGlobalApi::from_pre_peer_id_endpoint(SafeUrl::parse(&peer_env_vars.FM_API_URL)?);
+            endpoints.insert(*peer, peer_env_vars.FM_API_URL.clone());
             admin_clients.insert(*peer, admin_client);
             peer_to_env_vars_map.insert(peer.to_usize(), peer_env_vars);
         }
 
-        run_dkg(admin_clients, params).await?;
+        let fedimint_cli_version = crate::util::FedimintCli::version_or_default().await;
+        if fedimint_cli_version >= *VERSION_0_3_0_ALPHA {
+            run_cli_dkg(params, endpoints).await?;
+        } else {
+            // TODO: remove this after we drop support for v0.2.x
```

Comment:
nit: @bradleystachurski mentions we have a different comment style for things to-be-phased-out

Reply:
Let's update to

```
TODO(support:v0.2)
```

Similar to https://github.com/fedimint/fedimint/blob/311006b32fcf923a299e4854ee8e70d1f29d52f0/devimint/src/tests.rs#L272

Related discussion https://github.com/fedimint/fedimint/issues/4424